docs/changelog/1_0_rc2.rst

```diff
@@ -301,8 +301,9 @@ Server configuration
     - ``never``
 
 ``EDGEDB_SERVER_TLS_CERT_MODE``
-    - ``file == default``
-    - ``self_signed`` — generate self-signed on bootstrap and when expired
+    - ``require_file`` (``== default`` if EDGEDB_SERVER_SECURITY is ``strict``)
```

Comment:
What is default if security is not `strict`?

Reply:
Clarified.